AmazonAdvertisingApi/Client.php

```diff
@@ -96,286 +98,662 @@ public function listProfiles()
+    {
+        return $this->_operation(
+            "{$this->prefixes["products"]}campaigns/{$campaignId}", null, "DELETE"
+        );
+    }
+
+    public function listSponsoredProductsCampaigns($data = null)
+    {
+        return $this->_operation(
+            "{$this->prefixes["products"]}campaigns", $data
+        );
+    }
+
+    public function listSponsoredProductsCampaignsEx($data = null)
+    {
+        return $this->_operation(
+            "{$this->prefixes["products"]}campaigns/extended", $data
+        );
+    }
+
+    public function getSponsoredDisplayCampaign($campaignId)
+    {
+        return $this->_operation(
+            "{$this->prefixes["display"]}campaigns/{$campaignId}"
+        );
+    }
+
+    public function getSponsoredDisplayCampaignEx($campaignId)
+    {
+        return $this->_operation(
+            "{$this->prefixes["display"]}campaigns/extended/{$campaignId}"
+        );
+    }
+
+    public function createSponsoredDisplayCampaigns($data)
+    {
+        return $this->_operation(
+            "{$this->prefixes["display"]}campaigns", $data, "POST"
+        );
+    }
+
+    public function updateSponsoredDisplayCampaigns($data)
+    {
+        return $this->_operation(
+            "{$this->prefixes["display"]}campaigns", $data, "PUT"
+        );
+    }
+
+    public function archiveSponsoredDisplayCampaign($campaignId)
+    {
+        return $this->_operation(
+            "{$this->prefixes["display"]}campaigns/{$campaignId}", null, "DELETE"
+        );
+    }
+
+    public function listSponsoredDisplayCampaigns($data = null)
+    {
+        return $this->_operation(
+            "{$this->prefixes["display"]}campaigns", $data
+        );
+    }
+
+    public function listSponsoredDisplayCampaignsEx($data = null)
+    {
+        return $this->_operation(
+            "{$this->prefixes["display"]}campaigns/extended", $data
+        );
+    }
+
+    public function getSponsoredBrandsCampaign($campaignId)
+    {
+        return $this->_operation(
+            "{$this->prefixes["brands"]}campaigns/{$campaignId}"
+        );
+    }
+
+    public function createSponsoredBrandsCampaigns($data)
+    {
+        return $this->_operation(
+            "{$this->prefixes["brands"]}campaigns", $data, "POST"
+        );
+    }
+
+    public function updateSponsoredBrandsCampaigns($data)
+    {
+        return $this->_operation(
+            "{$this->prefixes["brands"]}campaigns", $data, "PUT"
+        );
+    }
+
+    public function archiveSponsoredBrandsCampaign($campaignId)
+    {
+        return $this->_operation(
+            "{$this->prefixes["brands"]}campaigns/{$campaignId}", null, "DELETE"
+        );
+    }
+
+    public function listSponsoredBrandsCampaigns($data = null)
+    {
+        return $this->_operation(
+            "{$this->prefixes["brands"]}campaigns", $data
+        );
+    }
+
+    public function getSponsoredProdcutsAdGroup($adGroupId)
+    {
```

Comment:
This and all the ones below are spelt prod**cu**ts

Reply:
oops :/